icons/icons.php

```diff
@@ -4,8 +4,8 @@
 
 function siteorigin_widgets_icon_families_filter( $families ){
 	$bundled = array(
+        'fontawesome' => __( 'Font Awesome', 'so-widgets-bundle' ),
```

Comment:
It's minor, but can you use tab indenting here instead of space indenting.

Reply:
Yes, np ! I missed that ;)